### ISSUE TITLE
Option to hide the Doorbell Automation button

### ISSUE THREAD

**Issue by CocoaBob** [Brandawg93/homebridge-nest-cam#64]
Hi,

In the recent update [v2.2.0](https://github.com/Brandawg93/homebridge-nest-cam/releases/tag/v2.2.0), a new button is added if the user has a Nest Hello doorbell. It's unconfigured by default, we can add actions to that button.

But the problem is that I don't know what I can do with that button, and I can't hide it if I don't need it.

I tried to use  it for turning on/off the video streaming, but it doesn't work well (and I have to use Nest app to reset the streaming).

So could you add an option to hide that button? I've disabled most of the features except audio, but the button is still there.

Thanks!

**Comment by CocoaBob** [comment]
Sorry, though I didn't find it in Config UI X but I just noticed that [v2.3.0](https://github.com/Brandawg93/homebridge-nest-cam/releases/tag/v2.3.0) has added the option to disable that button, thanks!

**Comment by CocoaBob** [comment]
Sorry, I forgot to reopen this issue, please have a check, thanks!
The description is in the above comment, I just tested the latest version v2.3.3, it's still there, even I uncheck the `Doorbell Switch` option.

**Comment by Brandawg93** [comment]
I've published a new test version that should fix this. You can install it via `npm -g i homebridge-nest-cam@test`. Let me know if this fixes it for you.

**Comment by CocoaBob** [comment]
Just tested it, it crashes.

```
[5/25/2020, 3:47:17 PM] [Nest-cam] Initializing Nest-cam platform...
[5/25/2020, 3:47:17 PM] TypeError: Cannot read property 'disableAudio' of undefined
    at new NestCamPlatform (/usr/lib/node_modules/homebridge-nest-cam/src/index.ts:78:59)
    at /usr/lib/node_modules/homebridge/src/server.ts:396:40
    at Array.forEach (<anonymous>)
    at Server.loadPlatforms (/usr/lib/node_modules/homebridge/src/server.ts:373:27)
    at Server.start (/usr/lib/node_modules/homebridge/src/server.ts:152:29)
    at cli (/usr/lib/node_modules/homebridge/src/cli.ts:80:10)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
[5/25/2020, 3:48:49 PM] Got SIGTERM, shutting down Homebridge...
```

Where should I set the `disableAudio` value? My config looks like this
```
{
    "googleAuth": {
        "issueToken": "",
        "cookies": "",
        "apiKey": ""
    },
    "ffmpegCodec": "libx264",
    "advancedSettings": {
        "motionDetection": true,
        "streamingSwitch": false,
        "doorbellAlerts": true,
        "doorbellSwitch": false,
        "disableAudio": false
    },
    "platform": "Nest-cam"
}
```

**Comment by Brandawg93** [comment]
I see what the issue is. I renamed the config on accident. Try renaming `advancedSettings` to `options` and see if it works.

**Comment by Brandawg93** [comment]
I've just released v2.3.4 which should fix this issue. Let me know if it doesn't.

**Comment by CocoaBob** [comment]
Great! Just installed v2.3.4, and it works like a charm! Thank you!
